R/aggregate_kicking_stats.R

```diff
@@ -0,0 +1,161 @@
+#' Summarize Kicking Stats
+#'
+#' @description Build columns that aggregate kicking stats at the game level.
+#'
+#' @param pbp A Data frame of NFL play-by-play data typically loaded with
+#' [load_pbp()] or [build_nflfastR_pbp()].
+#' @param weekly If `TRUE`, returns week-by-week stats, otherwise, stats for
+#' the entire Data frame.
+#'
+#' @examples
+#'
+#' \donttest{
+#' pbp <- nflreadr::load_pbp(2021)
+#' calculate_kicking_stats(pbp, weekly = TRUE)
+#' }
+#'
+#' @return a dataframe of kicking stats
+#' @seealso <https://nflreadr.nflverse.com/reference/load_player_stats.html> for the nflreadr function to download this from repo (`stat_type = "kicking"`)
+#' @export
+calculate_kicking_stats <- function(pbp, weekly = FALSE){
```

Comment:
similarly, `calculate_player_stats_kicking` to follow the `calculate_player_stats_def` naming

Reply:
Good eye--updated the function name!